src/main/java/Duke.java

```
@@ -1,10 +1,171 @@
+                    throw new FileNotFoundException();
+                }
+                if (isDone) {
+                    task.setDone();
+                }
+            }
+            s.close();
+        } catch (FileNotFoundException e) {
+            System.out.println("Unable to find file\n");
+        }
+    }
+
+    public static void inputTask() {
+        Scanner in = new Scanner(System.in);
+        String inputTask = in.nextLine();
+
+        while(!inputTask.equals("bye")) {
+            try{
+                if (inputTask.equals("list")) {
+                    System.out.println(LINE + "Here are the tasks in your list:\n");
+                    for (int i = 0; i < tasks.size(); i++) {
+                        System.out.println((i + 1) + ". " + tasks.get(i).toString());
+                    }
+                    System.out.println(LINE);
+                } else if (inputTask.contains("done")) {
+                    int sep = inputTask.indexOf(" ");
+                    int number = Integer.parseInt(inputTask.substring(5));
+                    markAsDone(number);
+                } else if (inputTask.contains("todo")) {
+                    addToDo(inputTask);
+                } else if (inputTask.contains("deadline")) {
+                    addDeadline(inputTask);
+                } else if (inputTask.contains("event")) {
+                    addEvent(inputTask);
+                } else if (inputTask.contains("delete")) {
+                    deleteTask(inputTask);
+                }
+                else {
+                    throw new DukeException();
+                }
+            } catch (DukeException e) {
+                System.out.println("☹ OOPS!!! I'm sorry, but I don't know what that means :-( ");
+                System.out.println(LINE);
+            }
+            inputTask = in.nextLine();
+        }
+        if (inputTask.equals("bye")) {
+            try {
+                writeToFile();
+            } catch (IOException e) {}
```

Avoid having an empty catch block. If you really want to leave empty, then leave some comment as to why it is left empty.